src/pages/api/__tests__/github-load-notes.test.ts

```diff
@@ -0,0 +1,15 @@
+import {parseMdxNotesFile} from '../github-load-notes'
+
+// this had to use an async/await because the mdx compiler
+// is set up to be async and return a promise
+test('properly parses a markdown text string', async () => {
+  const markdownNotes = `<TimeStamp start="0:44" end="0:50">
```

Comment:
our markdown files are "just strings" so you can simulate conditions here and check the results below"